presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/env/common/Standard.java

```diff
@@ -68,18 +69,21 @@
 
     private final String imagesVersion;
     private final File serverPackage;
+    private final boolean debug;
 
     @Inject
     public Standard(
             DockerFiles dockerFiles,
             PortBinder portBinder,
             EnvironmentConfig environmentConfig,
-            @ServerPackage File serverPackage)
+            @ServerPackage File serverPackage,
+            @Debug boolean debug)
```

Comment:
It would be nice if the "Disable Presto health checks" change was separate from the debug's refactor.